app/upgrades/v12/constants.go

```diff
@@ -18,9 +18,9 @@ package v12
 
 const (
 	// UpgradeName is the shared upgrade plan name for mainnet
-	UpgradeName = "v12.1.0"
```

Comment:
I think we don't need these changes on the `constants.go`, as it is a patch release

Reply:
I was wondering that, I'll revert these changes